backup/get_dropbox_tokens.py

```diff
@@ -0,0 +1,22 @@
+import os
+from dropbox import DropboxOAuth2FlowNoRedirect
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Is `dotenv` added to the requirements.txt?

Reply:
Also I believe https://github.com/metakgp/metakgp-wiki/blob/af9a33131ffc56d08bdfa1457cdb95c0c2407b08/backup/run_backup.sh#L8 this file loads the `.env` already. It would be better to do it in a single place (better to do in Python).

Reply:
Ideally, shouldn't the tokens already be part of the environment?

Reply:
I think we don't need to add  `get_dropbox_tokens.py` in the container. We only need to run it once to get the access and refresh tokens which needs to be filled in the .env and then the container will be built.

Reply:
Then I think it would be better to put this script in `/scripts/`

Reply:
Done. Also, the scripts now asks for input for app_key and app_secret.